modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -89,6 +89,8 @@ var Instrument = {
             }
             for (element of elements[i].Elements) {
                 switch (element.Type) {
+                    case "line":
+                        content += '<br>\n';
```

Comment:
The format for static content (including br tags) needs to be like the format of "label", including the static type and (empty) fieldname. I don't think this will load properly in LORIS as written. It needs to be

`static{@}{@}<br>\n`